stan/math/prim/scal/meta/rm_zeroing.hpp

```diff
@@ -0,0 +1,34 @@
+#ifndef STAN_MATH_PRIM_SCAL_META_RM_ZEROING_HPP
+#define STAN_MATH_PRIM_SCAL_META_RM_ZEROING_HPP
+
+namespace stan {
+namespace math {
+
+template <class T>
+struct zeroing;  // forward declare
+
+}  // namespace math
+
+/**
+ * rm_zeroing replaces <code>zeroing<T></code> with
+ * T. This is the general version of the trait.
+ */
+template <class T>
+struct rm_zeroing {
+  typedef T type;
+};
+
+/**
+ * rm_zeroing replaces <code>zeroing<t></code> with
+ * T. This is the removal version of the trait.
+ */
+template <class T>
+struct rm_zeroing<math::zeroing<T>> {
```

Comment:
remove `math::` in front of `zeroing`

Reply:
at that line in the file, the `math` sub-namespace is not open, just the outer `stan`, so if I did that, it would cause a compile error

Reply:
Thanks. I didn't catch that on the first read. Forgot that our meta is in the Stan namespace.

Reply:
Thanks. I didn't catch that on the first read. Forgot that our meta is in the Stan namespace.